.github/workflows/debian_10_arm.yml

```diff
@@ -98,9 +99,30 @@ jobs:
         -DENABLE_TESTS=ON \
         -DENABLE_PYTHON=OFF
 
+  CXX_Unit_Tests:
+    name: C++ unit tests
+    needs: [ Docker, Build, Smart_CI ]
+    uses: ./.github/workflows/job_cxx_unit_tests.yml
+    with:
+      runner: 'aks-linux-16-cores-arm'
+      image: ${{ fromJSON(needs.docker.outputs.images).ov_test.debian_10_arm }}
+      affected-components: ${{ needs.smart_ci.outputs.affected_components }}
+      os: 'debian_10'
+
+  CPU_Functional_Tests:
+    name: CPU functional tests
+    if: ${{ 'false' }} # Ticket: 153289
+    # if: fromJSON(needs.smart_ci.outputs.affected_components).CPU.test
+    needs: [ Docker, Build, Smart_CI ]
+    uses: ./.github/workflows/job_cpu_functional_tests.yml
+    with:
+      runner: 'aks-linux-16-cores-arm'
+      image: ${{ fromJSON(needs.docker.outputs.images).ov_test.debian_10_arm }}
+      python-version: '3.7'
```

Comment:
why 3.7? We support 3.9-3.12 only

Reply:
And why it is not a part of the docker image?

Reply:
> why 3.7
Because it is Debian 10's default Python version and if another one is specified instead of 3.7, the actions fails as the newer Python versions could be installed only from the source on Debian 10.

> And why it is not a part of the docker image?
It is. However, since it is a reusable job, some workflows (mac) do not have Docker images and rely on this action for Python setup.

Anyway, underneath it just checks if Python 3.7 is already installed and if so, skips the rest of the action.